docs/articles/documentation/extending-testcafe/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+title: Extending TestCafe
+permalink: /documentation/extending-testcafe/
+---
+# Extending TestCafe
+
+TestCafe is a powerful application, however you may need a specific functionality that TestCafe does not provide initially.
```

Comment:
> powerful application

I'm not sure if `application` is a correct word here. TestCafe can be used as command line tool or node module. Term `application` primarily used for the programs with user interface. Maybe `framework` or `library`? \cc @VasilyStrelyaev